src/udev/udev-builtin-net_id.c

```diff
@@ -35,10 +35,12 @@
  * Type of names:
  *   b<number>                             — BCMA bus core number
  *   c<bus_id>                             — CCW bus group name, without leading zeros [s390]
- *   o<index>[d<dev_port>]                 — on-board device index number
- *   s<slot>[f<function>][d<dev_port>]     — hotplug slot index number
+ *   o<index>[d<dev_port>][n<phys_port_name>]
+ *                                         — on-board device index number
+ *   s<slot>[f<function>][d<dev_port>][n<phys_port_name>]
+ *                                         — hotplug slot index number
  *   x<MAC>                                — MAC address
- *   [P<domain>]p<bus>s<slot>[f<function>][d<dev_port>]
+ *   [P<domain>]p<bus>s<slot>[f<function>][d<dev_port>][n<phys_port_name>]
```

Comment:
Hm, then this needs changing too:
`[P<domain>]p<bus>s<slot>[f<function>][n<phys_port_name>|d<dev_port>]` ?